database/dbConfig.js

```diff
@@ -0,0 +1,26 @@
+const knex = require('knex');
+
+const knexConfig = require('../knexfile.js');
+
+const db = knex(knexConfig.development);
+
+module.exports = {
+  
+insert: (user) =>{
+    return db('users').insert(user);
+},
+
+findByUsername: (username) =>{
+    return db('users').where('username', username).first();
```

Comment:
You can use an object for the argument of `where`:
```javascript
return db('users').where({ username: username }).first();
```
and, where I'm getting at, using shorthand syntax (since the variable name is the same as the key):
```javascript
return db('users').where({ username }).first();
```